packages/@aws-cdk/aws-synthetics/lib/canary.ts

```diff
@@ -290,35 +291,18 @@ export class Canary extends cdk.Resource {
    * @default avg over 5 minutes
    */
   public metricSuccessPercent(options?: MetricOptions): Metric {
-    return this.metric('SuccessPercent', options);
+    return this.cannedMetric(CloudWatchSyntheticsMetrics.successPercentAverage, options);
   }
 
   /**
    * Measure the number of failed canary runs over a given time period.
    *
-   * @param options - configuration options for the metric
+   * Default: sum over 5 minutes
    *
-   * @default avg over 5 minutes
-   */
-  public metricFailed(options?: MetricOptions): Metric {
-    return this.metric('Failed', options);
-  }
-
-  /**
-   * @param metricName - the name of the metric
    * @param options - configuration options for the metric
-   *
-   * @returns a CloudWatch metric associated with the canary.
-   * @default avg over 5 minutes
    */
-  private metric(metricName: string, options?: MetricOptions): Metric {
-    return new Metric({
-      metricName,
-      namespace: 'CloudWatchSynthetics',
-      dimensions: { CanaryName: this.canaryName },
-      statistic: 'avg',
-      ...options,
-    }).attachTo(this);
+  public metricFailed(options?: MetricOptions): Metric {
+    return this.cannedMetric(CloudWatchSyntheticsMetrics.failedSum, options);
```

Comment:
`!` It looks like you've changed the default statistic of this from `avg` to `sum`.

Reply:
I did, mostly because this webpage says:

> Valid Statistic: Sum

https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_metrics.html

You are correct that it probably should have been a separate PR, though :(

Reply:
Ah, I see. So this is another case where the current behavior would never yield any data and is useless? If that's the case, I am less alarmed about it; however, a separate PR -- just so it gets called out in the CHANGELOG -- would probably be _slightly_ better.

Reply:
Not "never yield any data"... just "yield useless data".

If data points under this metric get emitted as `1` on failures, and *nothing* (as opposed to `0`) on success, then `Avg` will just always return `1`--whereas the `Sum` will return the actual count over a given time period.